docs/advanced/command-line/files/wrangle-properties.md

```diff
@@ -0,0 +1,152 @@
+# Wrangle Properties
+In the command line version, the wrangle properties in the control panel tab in the web browser detailed in the [Steering Section](../../../../guides/refining-solutions/steering/) can also be specified in a properties file, eg. **wrangle.properties** and passed to Data Preparer using the -p command line option. 
+> Note: We use .properties as our naming convention, but you are free to use whatever name you like.
+
+## Format of the file
+We have provided an example of how the contents of the file should be structured in **wrangle1.properties** and **wrangle2.properties** in the Manual Books example.
+The properties in this file are expected to be as described below:
+
+### General Settings
+- workflow. The system components that will participate in the wrangling process. This
+is a comma-separated list, containing any of the following: match, repair, transform, profile, map, select. Note that some of these components require the output of others, as follows:
+    - repair requires match; 
+    - transform requires match;
+    - map requires match, profile; 
+    - and select requires map, match, profile.
+
+    > Note: A required component will be executed, even if it is omitted here. For instance, defining a workflow as map, select, will trigger execution of
+the workflow match, profile, map, select, because match and profile are required by map. 
+The default workflow includes all components: match, repair, transform, profile, map, select.
+
+
+- workflow.incremental. Setting the value of this property to no, means that when a new wrangling process is executed, all components will be executed. Setting it to yes, means that when a new wrangling process is executed, any existing results will be reused i.e. components whose configuration hasn’t changed since the last run will not run again. 
+> The default value is yes.
```

Comment:
Note: Default value is yes

Reply:
I'll stick to not adding note for defaults

Reply:
because that seems to be the style